pyro/infer/predictive.py

```diff
@@ -54,14 +56,30 @@ def _predictive(model, posterior_samples, num_samples, return_sites=None,
         sample = sample.reshape((num_samples,) + (1,) * (max_plate_nesting - len(sample_shape)) + sample_shape)
         reshaped_samples[name] = sample
 
+    def _vectorized_fn(fn):
```

Comment:
Why did you revert this?

Reply:
Oppss... I didn't merge dev branch yet.

Reply:
Hmm, this is an issue of IDE I guess... I will change it back

Reply:
I see the reason now. I make the change in another branch, then copy that file into this branch, then this kind of thing happens...

Reply:
Thanks, I'll try it next time.